sdk/python/test_data/pipelines/parallelfor_fan_in/nested_with_artifacts.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
```

Comment:
I would add these only at the time we support fan-in artifacts. At this moment, this are dead code that serves no real purpose.

Reply:
Agreed. Updated.